src/test/java/jalau/at18/katas/likes/daniela/LikesTest.java

```diff
@@ -0,0 +1,31 @@
+package jalau.at18.katas.likes.daniela;
+
+import static org.junit.Assert.*;
+import org.junit.Test;
+
+public class LikesTest {
+    @Test
+    public void testLikes() {
+        Like like = new Like();
+        String[] finalnamecase1 = {"Daniela", "Alejandra", "javier", "mayra"};
+        String[] finalnamecase2 = {"Daniela", "Alejandra", "javier"};
+        String[] finalnamecase3 = {"Daniela", "Alejandra"};
+        //String[] finalnamecase4 = {"Daniela"};
+        //String[] finalnamecase5 = {};
```

Comment:
`finalnamecase4` and `finalnamecase5` should be verified as well, why were those commented?